src/status_im2/contexts/onboarding/intro/events.cljs

```diff
@@ -0,0 +1,12 @@
+(ns status-im2.contexts.onboarding.intro.events
+  (:require [utils.re-frame :as rf]
+            [status-im2.contexts.onboarding.common.overlay.view :as overlay]
+            [status-im2.contexts.profile.profiles.view :as profiles]))
+
+(rf/defn intro-on-focus
+  {:events [:onboarding/intro-on-focus]}
+  [_]
+  (when @overlay/blur-dismiss-fn-atom
```

Comment:
this is a ratom right? can't we pass this as a prop instead? 🤔

Reply:
im trying to do my best, entire navigation is broken now, im just trying to fix it step by step

Reply:
Yes I agree, navigation needs a lot of work. Perhaps we can start to improve this soon and figure what are the issues.

But still I don't see how using a global ratom inside a function is helping? Is it not possible to the same yet pass the ratom as a prop so at least this function remains pure ?

Reply:
yes its a wrong implementation,its not possible to implement it without atom, but in proper implementation there should be no atom at all